src/Shared/TaskHostConfiguration.cs

```diff
@@ -417,20 +425,27 @@ public void Translate(ITranslator translator)
             translator.TranslateCulture(ref _culture);
             translator.TranslateCulture(ref _uiCulture);
 #if FEATURE_APPDOMAIN
-            byte[] appDomainConfigBytes = null;
 
-            // Set the configuration bytes just before serialization in case the SetConfigurationBytes was invoked during lifetime of this instance.
-            if (translator.Mode == TranslationDirection.WriteToStream)
+            // Skip AppDomain configuration when targeting .NET Task Host (Runtime="Net").
```

Comment:
Is there a list of all the runtime names and their meaning?

Reply:
not sure, @rainersigwald might know more.